### PR TITLE
feat(accordion): add basic structure

### PR DESCRIPTION
**Pull request by marcoswebdev**
Basic structure created

### CODE REVIEW

File: packages/accordion/README.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+  </lion-accordion>
+`;
+```
+
+## How to use
+
+### Installation
+
+```bash
+npm i --save @lion/accordion;
```
Comment (gerjanvangeest) [inline packages/accordion/README.md:43]: No need for `;` (i see its copied of tabs)

File: packages/accordion/README.md [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+    labore reiciendis saepe, iure, optio officiis obcaecati quibusdam.
+  </p>
+</lion-accordion>
+```
+
+## Examples
+
+### Expanded
+
+You can set `expanded` to pre-expand a certain invoker.
```
Comment (gerjanvangeest) [inline packages/accordion/README.md:76]: Can we add a small text about the heading/button structure? Something like:

> An accordion exists off a list of expandable headings (of the same level). To get this behavior you need to add a `slot="invoker"` to the heading and place a button as the content.

File: packages/accordion/README.md (hunk 1 above)
Comment (marcoswebdev) [reply packages/accordion/README.md]: Done!

File: packages/accordion/README.md (hunk 2 above)
Comment (marcoswebdev) [reply packages/accordion/README.md:78]: Done!

**Pull request merged**